pkg/state/conformance/state.go

```diff
@@ -548,11 +546,17 @@ func (suite *StateSuite) TestWatchFor() {
 	wg.Wait()
 
 	suite.Require().NoError(err)
-	suite.Assert().Equal(r.Metadata().String(), path1.Metadata().String())
+
+	suite.Assert().Equal(r.Metadata().Type(), path1.Metadata().Type())
```

Comment:
not sure why this has to be changed now?

Reply:
It's to compare without the version - type, ns, id but not version

Reply:
version should match as well!